selfdrive/controls/lib/lateral_planner.py

```diff
@@ -103,7 +105,7 @@ def publish(self, sm, pm):
     lateralPlan.modelMonoTime = sm.logMonoTime['modelV2']
     lateralPlan.dPathPoints = self.y_pts.tolist()
     lateralPlan.psis = self.lat_mpc.x_sol[0:CONTROL_N, 2].tolist()
-    lateralPlan.curvatures = self.lat_mpc.x_sol[0:CONTROL_N, 3].tolist()
+    lateralPlan.curvatures = (self.lat_mpc.x_sol[0:CONTROL_N, 3]/sm['carState'].vEgo).tolist()
```

Comment:
@YassineYousfi this is throwing RuntimeWarnings for dividing by 0 on startup

Reply:
fixing